recipes/jupyter-jsmol/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "jupyter-jsmol" %}
+{% set version = "2022.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_jsmol-{{ version }}.tar.gz
+  sha256: 6889b236dcd1eb69093ec1bc87ce352809c89a551214eada81186c2830c07c9a
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=40.8.0
+    - jupyterlab >=3.0.0
```

Comment:
I *think* you should be good with only the `jupyter-packaging`.
```suggestion
```

Reply:
@dopplershift Thanks for the suggestion. Unfortunately, the `win_64 win` pipeline wails without it because of the `jupyter labextension build .` command during the build process